docs/docs/new-features.md

```
@@ -303,6 +303,7 @@ Modified/added events
 - `allowOverrideClientSettingsOnCreateCall=false` added
 - `sessionsCleanupDelayInMinutes=60` added
 - `graphqlWebsocketUrl=${bigbluebutton.web.serverURL}/graphql` added
+- `muteOnStart` default value set to `true` - which helps now that `transparentListenOnly` is enabled by default too. See (PR 20848)[https://github.com/bigbluebutton/bigbluebutton/issues/20848] for more info.
```

Markdownlint (0.37.0)

(PR 20848)[https://github.com/bigbluebutton/bigbluebutton/issues/20848]
Reversed link syntax

(MD011, no-reversed-links)